cmd/trace/main.go

```diff
@@ -105,7 +104,26 @@ func failWithMessage(msg string) {
 }
 
 func printHelp() {
-	fmt.Fprintf(os.Stderr, `Usage: kubectl crossplane trace TYPE[.GROUP] NAME [-n| --namespace NAMESPACE] [-h|--help]
+	fmt.Fprintf(os.Stderr, `Trace a Crossplane resource
+
+  Traces a given Crossplane resource and gathers information of relevant resources. Displayed information contains 
+details of each resource status as well as an overview summarizing the overall state.
+
+  By specifying the output format as 'dot', you can obtain a graph definition in dot format which can be vizualized with
+GraphViz.
+
+  Each resource in the trace output categorized in one of the following states which is represented in text (state 
+column of overview) and graph output (as node attributes) accordingly.
+ 
+ -------------------------------------------------------
+|   State  | Text Representation | Graph Representation | 
+|----------|---------------------|----------------------|
+| Ready    |          ✓          |     Solid - Black    |
+| NotReady |          !          |    Solid - Orange    |
+| Missing  |          ✕          |     Dotted - Red     |
+ ------------------------------------------------------- 
+
+Usage: kubectl crossplane trace TYPE[.GROUP] NAME [-n| --namespace NAMESPACE] [-h|--help]
```

Comment:
We should put the usage line at the top so it matches the way manpages order things

Reply:
Well actually, I preferred this way to be inline with kubectl commands, i.e. `kubectl get --help`
What I see is `man` shows top section of the message first, but when one writes `--help` it is different. 
What do you think?

Reply:
Ahh, ok, that makes sense. I didn't realize `kubectl` does it that way for `--help`, and I like the idea of matching `kubectl`. This looks good to me then, so I'll approve it!